server/src/objects/parachute.ts

```diff
@@ -70,6 +72,25 @@ export class Parachute extends BaseGameObject<ObjectCategory.Parachute> {
                                 amount: Infinity,
                                 source: crate
                             });
+                            if (object.definition.idString == "airdrop_crate_locked" && object != crate){
```

Comment:
checking for idStrings is bad but this may be the only option so add a comment saying it is a hack